internal/provider/resource_streaming_events.go

```diff
@@ -0,0 +1,131 @@
+package provider
+
+import (
+	"context"
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/packetfabric"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const VCSDescription = "Specific logical interfaces you wish to subscribe to, each identified by a combination of the virtual circuit ID and port circuit ID " +
```

Comment:
why add the description as a constant and not directly in the schema?

Reply:
The text seems too long.